### PR TITLE
Kubernetes 1.15.5 release

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/6431

### CODE REVIEW

File: v_4_9_0/cloudconfig.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package v_4_8_0
+package v_4_9_0
```
Comment (MarcelMue) [inline v_4_9_0/cloudconfig.go]: This should be done in a separate PR. So the adding of `4_9_0` is separate from the actual change.

Comment (tfussell) [reply v_4_9_0/cloudconfig.go]: So three PRs: copy previous version, rename package, populate changes?

Comment (MarcelMue) [reply v_4_9_0/cloudconfig.go]: Yes - so it's a bit more easy to see which the actual changes are. Not much but a little :P

Comment (tfussell) [reply v_4_9_0/cloudconfig.go]: Heh, okay. I did do it in separate commits, but I will revert and split into another PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,16 +9,21 @@ The latest version is considered WIP and it is a subject of change. All other
...
-### Added
-
-- Bind kube-proxy metrics address to 0.0.0.0 instead on default 127.0.0.1 in config file.
+## [v4.9.0] - 2019-10-17
 
 ### Changed
 
+- Bind kube-proxy metrics address to 0.0.0.0 instead of default 127.0.0.1 in config file.
 - Remove Calico Node limits.
+- Update Kubernetes to `1.15.4` (CVE-2019-11251).
```
Comment (stone-z) [inline CHANGELOG.md]: 1.15.5 now I think

**Pull request merged**